API.md

```diff
@@ -72,22 +72,30 @@ Returns a single election with some more data than the list:
 {
   id: string,
   createdAt: datetime,
-  shortIdentifier: string,
+  electionHash: string,
```

Comment:
I think I actually prefer `shortIdentifier` here. Mostly, I just want the names to be clear about their intent. Is this for cryptographic security? For end-user disambiguation? Something else?

Reply:
It is for both. I tend to agree with you that intent is better, but on this one I'm feeling like this gives us more flexibility in terms of how we use it. It is mostly disambiguation.

Reply:
This is full SHA256 hash of election JSON. We will slice upon display.